ocp_resources/csi_storage_capacity.py

```diff
@@ -0,0 +1,67 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class CSIStorageCapacity(NamespacedResource):
+    """
+    CSIStorageCapacity object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/storage_apis/storage-apis-index.html#csistoragecapacity-storage-k8s-iov1
```

Comment:
ditto (https://kubernetes.io/docs/reference/kubernetes-api/config-and-storage-resources/csi-storage-capacity-v1/#CSIStorageCapacity)

Reply:
Done and thank you!